src/bilder/images/edxapp/templates/edxapp/mitxonline/common_values.yml.tmpl

```diff
@@ -425,17 +425,19 @@ REGISTRATION_EXTRA_FIELDS:
 RETIRED_EMAIL_DOMAIN: retired.invalid
 RETIRED_EMAIL_PREFIX: retired__user__
 RETIRED_USERNAME_PREFIX: retired__user__
-RETIREMENT_SERVICE_WORKER_USERNAME: retirement_worker
+RETIREMENT_SERVICE_WORKER_USERNAME: retirement_service_worker
 RETIREMENT_STATES:
     - PENDING
-    - LOCKING_ACCOUNT
-    - LOCKING_COMPLETE
     - RETIRING_FORUMS
     - FORUMS_COMPLETE
-    - RETIRING_EMAIL_LISTS
-    - EMAIL_LISTS_COMPLETE
```

Comment:
Why are we removing retiring email lists? Do you know which email lists this refers to?

Reply:
> Why are we removing retiring email lists?

The endpoint for retiring email lists is no longer there, It was removed under https://github.com/openedx/edx-platform/pull/19320 & had some changes to manage this through signals under https://github.com/openedx/edx-platform/pull/18423. So, RETIRING_LMS_MISC will do this for us moving forward. 

> Do you know which email lists this refers to?

It will create an `Optout` row for any courses they may be enrolled in. Emails for the enrolled courses are disabled for the user.